src/monsters.cpp

```diff
@@ -186,6 +239,175 @@ void MonsterType::createLoot(Container* corpse)
 	corpse->startDecaying();
 }
 
+void MonsterType::clone(MonsterType* root, MonsterType* child, const std::string& type) {
+	if (type == "loot") {
+		if (!child->lootItems.empty()) {
```

Comment:
Use std::list::assign(Iterator begin, Iterator end) to explicitly copy the containers.